examples/dhf4-with-tests/build.gradle

```diff
@@ -0,0 +1,40 @@
+buildscript {
+    repositories {
+        jcenter()
+        maven {url 'http://distro.marklogic.com/nexus/repository/maven-releases/'}
+    }
+    dependencies {
+        classpath "com.marklogic:ml-data-hub:4.1.0-rc1"
+    }
```

Comment:
I guess this will have to be changed to 4.1.0 after release

Reply:
Yeah I just marked this as WIP - I'm going to update it as I test out new bug fixes.